python/ambassador/ir/ir.py

```diff
@@ -426,12 +429,19 @@ def agent_finalize(self, aconf) -> None:
         # routed insecure. !*@&#*!@&#* We need per-mapping security settings.
         #
         # XXX What if they already have a mapping with this name?
+
+        ctx_name = None
+
+        if self.agent_orgination_ctx:
+            ctx_name = self.agent_origination_ctx.name
```

Comment:
These are spelled differently. Or my eyes are _really_ bad today.

Reply:
You were right, as it happens. FFS.